service/controller/clusterapi/v27/cloudconfig/legacy.go

```diff
@@ -0,0 +1,18 @@
+package cloudconfig
+
+import (
+	g8sv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	cmav1alpha1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
+)
+
+func cmaClusterToG8sCluster(cr cmav1alpha1.Cluster) g8sv1alpha1.Cluster {
+	return g8sv1alpha1.Cluster{
+		// TODO
+	}
+}
+
+func cmaClusterToG8sConfig(cr cmav1alpha1.Cluster) g8sv1alpha1.AWSConfigSpec {
+	return g8sv1alpha1.AWSConfigSpec{
+		// TODO
```

Comment:
Note that the mapping from CMA types to G8s types is missing here. We need this to get the correct information into the cloud config templates. I will do this in a follow up PR separately.